Rocket.Unturned/Chat/UnturnedChat.cs

```diff
@@ -116,11 +131,28 @@ public static void Say(IRocketPlayer player, string message, Color color, bool r
             }
         }
 
+        public static void Say(IRocketPlayer player, string message, Color color, string iconURL, bool rich)
+        {
+            if (player is ConsolePlayer)
+            {
+                Core.Logging.Logger.Log(message, ConsoleColor.Gray);
+            }
+            else
+            {
+                Say(new CSteamID(ulong.Parse(player.Id)), message, color, iconURL);
```

Comment:
The `rich` parameter is defined but not used, so rich text will not work. Same issue as I fixed in #3.

Reply:
oh thanks didn't see that, fixed